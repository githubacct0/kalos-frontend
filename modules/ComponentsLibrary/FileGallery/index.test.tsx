import React from 'react';
import renderer from 'react-test-renderer';
import { FileGallery } from './index';

test('FileGallery renders correctly', () => {
  const tree = renderer
    .create(
      <FileGallery
        loggedUserId={8418}
        title="Add Photo from Gallery"
        bucket="testbuckethelios" //"kalos-pre-transaction"
        onClose={() => console.log('close')}
        onAdd={console.log}
        removeFileOnAdd={false}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot(); // Rendered correctly
});
