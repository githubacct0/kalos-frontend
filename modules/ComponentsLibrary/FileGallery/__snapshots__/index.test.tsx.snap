// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileGallery renders correctly 1`] = `
<div>
  <div
    className="SectionBar fixedActions sticky"
  >
    <div
      className="SectionBarHeaderWrapper"
    >
      <div
        className="SectionBarHeader"
      >
        <div
          className="SectionBarTitleWrapper"
        >
          <h5
            className="MuiTypography-root SectionBarTitle MuiTypography-h5"
            onClick={[Function]}
          >
            Add Photo from Gallery
             
          </h5>
        </div>
      </div>
      <div
        className="SectionBarActions"
      >
        <div
          className="Actions"
        >
          <div
            className="ActionsActions"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained ButtonWrapper ActionsButton size-small status-undefined variant-contained MuiButton-containedPrimary MuiButton-containedSizeSmall MuiButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Close
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="InfoTable"
  >
    <div
      className="InfoTableHeader"
    >
      <div
        className="MuiTypography-root InfoTableColumn MuiTypography-body1"
        style={
          Object {
            "flexGrow": 0,
            "flexShrink": 1,
            "width": "33.333333333333336%",
          }
        }
      >
        <span
          className="InfoTableDir"
          style={
            Object {
              "cursor": "default",
              "justifyContent": "flex-start",
            }
          }
        >
          Photo
           
        </span>
      </div>
      <div
        className="MuiTypography-root InfoTableColumn MuiTypography-body1"
        style={
          Object {
            "flexGrow": 0,
            "flexShrink": 1,
            "width": "33.333333333333336%",
          }
        }
      >
        <span
          className="InfoTableDir"
          style={
            Object {
              "cursor": "default",
              "justifyContent": "flex-start",
            }
          }
        >
          Name
           
        </span>
      </div>
      <div
        className="MuiTypography-root InfoTableColumn MuiTypography-body1"
        style={
          Object {
            "flexGrow": 0,
            "flexShrink": 1,
            "width": "33.333333333333336%",
          }
        }
      >
        <span
          className="InfoTableDir"
          style={
            Object {
              "cursor": "default",
              "justifyContent": "flex-start",
            }
          }
        >
          Uploaded at
           
        </span>
      </div>
    </div>
    <div
      className="InfoTableRow hoverable"
    >
      <p
        className="MuiTypography-root InfoTableNoEntries MuiTypography-body1"
      >
        No entries found.
      </p>
    </div>
  </div>
</div>
`;
